app/src/main/java/com/celzero/bravedns/net/go/GoVpnAdapter.java

```diff
@@ -426,19 +396,19 @@ private static ParcelFileDescriptor establishVpn(BraveVPNService vpnService) {
                     .setSession("RethinkDNS")
                     .setMtu(VPN_INTERFACE_MTU)
                     .addAddress(LanIp.GATEWAY.make(IPV4_TEMPLATE), IPV4_PREFIX_LENGTH);
-            if(HomeScreenActivity.GlobalVariable.INSTANCE.getBraveMode() == DNS_MODE){
-                builder.addRoute(LanIp.DNS.make(IPV4_TEMPLATE),32);
+            if (HomeScreenActivity.GlobalVariable.INSTANCE.getBraveMode() == APP_MODE_DNS) {
+                builder.addRoute(LanIp.DNS.make(IPV4_TEMPLATE), 32);
                 builder.addDnsServer(LanIp.DNS.make(IPV4_TEMPLATE));
-            }else if(HomeScreenActivity.GlobalVariable.INSTANCE.getBraveMode() == FIREWALL_MODE){
-                builder.addRoute("0.0.0.0",0);
-            }else{
+            } else if (HomeScreenActivity.GlobalVariable.INSTANCE.getBraveMode() == APP_MODE_FIREWALL) {
+                builder.addRoute(DEFAULT_ROUTE_IP, 0);
```

Comment:
rename it to `UNSPECIFIED_IP` and `0` to `UNSPECIFIED_PORT`

Reply:
Modified.

Reply:
Modified.